lcio/trackerrawdata.go

```diff
@@ -19,10 +19,11 @@ type TrackerRawDataContainer struct {
 }
 
 type TrackerRawData struct {
-	CellID0 int32
-	CellID1 int32
-	Time    int32
-	ADCs    []uint16
+	CellID0    int32
+	CellID1    int32
+	Time       int32
+	nADCValues int32
```

Comment:
Of course, you are right.  This is my inexperience showing.  As you show below, the slices should be used the Go way.  Thank you!